core/sts-3.3.md

```diff
@@ -274,7 +311,7 @@ server may be offered at a different domain name, for example a subdomain.
 
 It's possible for an attacker to strip the STS `port` value from an initial connection
 established via an insecure connection, before the policy has been cached by the client.
-This represents a Bootstrap MITM (man-in-the-middle) vulnerability.
+This represents a bootstrap MITM (man-in-the-middle) vulnerability.
```

Comment:
"MITM" is explained in this part, but not in line 236. Since 236 is earlier it would make sense to move the `(man-in-the-middle)` up there or remove it completely.